### PR TITLE
OCP 4.17 test runs update 

### PR DESCRIPTION
**Pull request by aviavraham**

### CODE REVIEW

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 1]
```diff
@@ -589,3 +589,179 @@ periodics:
...
+    org: jboss-eap-qe
+    repo: openshift-eap-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: eap-ocp4.17-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.17-lp-interop-eap-74-xp-interop-aws
```
Comment (amp-rh) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:697]:
```suggestion
  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.17-lp-interop-eap-74-xp-interop-aws
    reporter_config:
    slack:
      channel: '#eap-ocp-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :success: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :dumpster-fire:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'
```

**Comment by aviavraham** [comment]
I created a new PR due to too many conflicts

**Pull request closed without merging**